target/test-classes/application.properties

```diff
@@ -0,0 +1,24 @@
+# Application properties
```

Comment:
We never commit. target folder in the commit. Target folder gets generated every time we do mvn clean install. So anything that gets generated need not be committed. It unnecessarily will occupy space on server

Reply:
Remove the target folder